docs/.vuepress/components/Button.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <!-- <h1>Test</h1> -->
+  <router-link v-if="isRouter" :to="to" :class="buttonClasses">
+    <slot />
+  </router-link>
+  <a href v-else-if="isA" :class="buttonClasses" :target="target" :rel="rel">
+    <slot />
+  </a>
+</template>
+
+<script>
+export default {
+  name: 'Button',
+  props: {
+    isA: {
+      type: Boolean,
+      default: true
+    },
+    isRouter: {
+      type: Boolean,
+      default: false
+    },
+    to: {
+      type: String
+    },
+    newTab: {
```

Comment:
I don't think this prop is needed either. If the link is external (e.g. `isExternal` is true), then we can add the `target="_blank" rel="noopener noreferrer"`